src/NosnocSolver.m

```diff
@@ -327,6 +326,11 @@ function  print_solver_stats(obj, results, stats)
             % TODO: Possible issue raise to casadi: allow unknown fields in options passed
             if strcmp(solver_options.solver, 'ipopt')
                 opts_casadi_nlp = rmfield(solver_options.opts_casadi_nlp, 'snopt');
+                if solver_options.timeout_cpu
+                    opts_casadi_nlp.ipopt.max_cpu_time = solver_options.timeout_cpu;
+                elseif solver_options.timeout_wall
+                    opts_casadi_nlp.ipopt.max_wall_time = solver_options.timeout_wall;
```

Comment:
should'nt   `opts_casadi_nlp.ipopt.` be   `opts_casadi_nlp.snopt.`

Reply:
No this is inside the if statement for ipopt not snopt